client/post-editor/editor-slug/index.jsx

```diff
@@ -83,6 +85,14 @@ const PostEditorSlug = React.createClass( {
 		}
 	},
 
+	componentWillReceiveProps( nextProps ) {
+		if ( nextProps.isVisible ) {
+			defer( () => {
+				this.focusSlug();
+			} );
```

Comment:
we don't need to use the brackets, as the fat-arrow syntax will just return the value form the function call if they are omitted.

```js
if ( nextProps.isVisible ) {
	defer( () => this.focusSlug() );
}
```